src/main/java/uk/ac/ebi/eva/submission/controller/SubmissionController.java

```diff
@@ -32,42 +32,88 @@ public SubmissionController(SubmissionService submissionService, WebinTokenServi
+    public ResponseEntity<?> markSubmissionUploaded(@RequestHeader("Authorization") String bearerToken,
+                                       @PathVariable("submissionId") String submissionId) {
+        String userId = this.getUserId(bearerToken);
+        if (Objects.isNull(userId)) {
+            return new ResponseEntity<>("Unauthorized", HttpStatus.UNAUTHORIZED);
+        }
+        // TODO: Confirm that this userId has access to the submission
+        Submission submission = this.submissionService.markSubmissionUploaded(submissionId);
+        if (Objects.nonNull(submission)) {
+            return new ResponseEntity<>(submission, HttpStatus.OK);
+        }else {
+            return new ResponseEntity<>("Not found", HttpStatus.NOT_FOUND);
+        }
     }
 
     @GetMapping("submission/{submissionId}/status")
-    public String getSubmissionStatus(@PathVariable("submissionId") String submissionId) {
-        return submissionService.getSubmissionStatus(submissionId);
+    public ResponseEntity<?> getSubmissionStatus(@RequestHeader("Authorization") String bearerToken,
+                                      @PathVariable("submissionId") String submissionId) {
+        String userId = this.getUserId(bearerToken);
+        if (Objects.isNull(userId)) {
+            return new ResponseEntity<>("Unauthorized", HttpStatus.UNAUTHORIZED);
+        }
+        // TODO: Confirm that this userId has access to the submission
+        String submissionStatus = submissionService.getSubmissionStatus(submissionId);
+        if (Objects.nonNull(submissionStatus)) {
+            return new ResponseEntity<>(submissionStatus, HttpStatus.OK);
+        }else {
+            return new ResponseEntity<>("Not found", HttpStatus.NOT_FOUND);
+        }
     }
 
     // TODO: admin end point (should not be exposed to the user)
     @PutMapping("submission/{submissionId}/status/{status}")
-    public Submission markSubmissionStatus(@PathVariable("submissionId") String submissionId, @PathVariable("status") SubmissionStatus status) {
-        return submissionService.markSubmissionStatus(submissionId, status);
+    public ResponseEntity<?> markSubmissionStatus(@RequestHeader("Authorization") String bearerToken,
+                                           @PathVariable("submissionId") String submissionId,
+                                           @PathVariable("status") SubmissionStatus status) {
+        String userId = this.getUserId(bearerToken);
+        if (Objects.isNull(userId)) {
+            return new ResponseEntity<>("Unauthorized", HttpStatus.UNAUTHORIZED);
+        }
+        // TODO: Confirm that this userId has access to the submission
+        Submission submission = this.submissionService.markSubmissionStatus(submissionId, status);
+        if (Objects.nonNull(submission)) {
+            return new ResponseEntity<>(submission, HttpStatus.OK);
+        }else {
+            return new ResponseEntity<>("Not found", HttpStatus.NOT_FOUND);
```

Comment:
**Optional**: Refactor User Id checking and subsequent responseentity creation into a separate method because it seems to have repeated a few times...